pkgs/by-name/wa/warp/package.nix

```diff
@@ -18,39 +18,28 @@
 , libadwaita
 , zbar
 , gst_all_1
-, Security
-, Foundation
 }:
 
 stdenv.mkDerivation rec {
   pname = "warp";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
     owner = "World";
     repo = "warp";
     rev = "v${version}";
-    hash = "sha256-GRxZ3y1PdJpBDnGCfmOmZgN8n1aaYf9IhyszRwo3MjQ=";
+    hash = "sha256-BUCkENpL1soiYrM1vPNQAZGUbRj1KxWbbgXR0575zGU=";
   };
 
-  patches = [
-    # https://gitlab.gnome.org/World/warp/-/merge_requests/74
-    (fetchpatch2 {
-      name = "rust-1.80-compat.patch";
-      url = "https://gitlab.gnome.org/World/warp/-/commit/38747cc2dde79089df53fd8451ea2db13f9f3714.patch";
-      hash = "sha256-9P5LwCHaC6J5WR2OnjCaNE+4de/Jv6XGXS7bOfYrM7w=";
-    })
-  ];
-
   postPatch = ''
     patchShebangs build-aux
   '';
 
-  cargoDeps = rustPlatform.fetchCargoTarball {
-    inherit src patches;
+  cargoDeps = rustPlatform.fetchCargoVendor {
```

Comment:
@TomaSajt Thanks for `fetchCargoVendor`! For some reason `fetchCargoTarball` fails to copy the `license-list-data` directory from the license crate.